csharp/.gitignore

```diff
@@ -329,3 +332,5 @@ ASALocalRun/
 .mfractor/
 
 deps/
+
+.DS_Store
```

Comment:
You have this in two places now :)

Reply:
Also, I can't comment on it but you've got a .DS_Store committed already. (Github is displaying it below this one). You need to remove it from the commit/staging to make the .gitignore apply.

Reply:
shiiiii